.github/workflows/complexity-check.yml

```
@@ -23,108 +23,108 @@ jobs:
-        # Check for oversized test files
-        echo "🔍 Checking individual test file sizes..."
-        OVERSIZED_FILES=""
-        while IFS= read -r file; do
-          if [ -n "$file" ]; then
-            LINE_COUNT=$(wc -l < "$file")
-            if [ "$LINE_COUNT" -gt 200 ]; then
-              echo "⚠️  Large test file: $file ($LINE_COUNT lines)"
-              OVERSIZED_FILES="$OVERSIZED_FILES\n  $file ($LINE_COUNT lines)"
-            fi
+          echo "🔍 Analyzing test infrastructure complexity..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)